test/helpers.html

```diff
@@ -62,8 +62,8 @@
     return grid.shadowRoot.querySelector('[slot="' + slotName + '"]');
   }
 
-  function getCellContent(grid, cell) {
-    cell = getCell(grid, cell);
+  function getCellContent(grid, _cell) {
```

Comment:
_ predix in parameter name?

Reply:
As you can see, we are setting `cell` variable inside the function itself, so we should not have the same names for function's parameter and internal variable.

Reply:
Ok. _ prefix is ok.